PySoar/importClass.py

```diff
@@ -13,24 +14,30 @@ class SoaringSpotImport(object):
     def __init__(self, url, download_progress):
```

Comment:
The current structure with a SoaringSpotImport is not very suitable for extension to other sites. It might be a good idea to generalize something like a 'ImportClass' which in turn calls specif SoaringSpot / StrePla functions.